File: Assignment 01/Program 12/prog12.c
//Write a Python program to append a new item to the end of the array.

#include<stdio.h>
#include<stdlib.h>

int main(){
    int n;
    printf("Enter the number of inputs: ");
    scanf("%d",&n);
    int *arr = (int*)malloc(n*sizeof(int));
    for (int i = 0; i < n; i++)
    {
        printf("Enter the number %d: ",i+1);
        scanf("%d",(arr+i));
    }

    int new_ele;
    printf("Enter the new element: ");
    scanf("%d",&new_ele);

    arr = (int*)realloc(arr,(n+1)*sizeof(int));
    arr[n] = new_ele;

    for (int i = 0; i < n+1; i++)
    {
        printf("%d ",*(arr+i));
    }
    free(arr);
    return 0;
}

File: Assignment 01/Program 12/prog12.py
#Write a Python program to append a new item to the end of the array.

import array as arr

a1 = arr.array('i')
n = int(input("Enter the number of inputs: "))
for i in range(n):
    val = int(input("Enter the element: "))
    a1.append(val)

new_ele = int(input("Enter the new element : "))
a1.append(new_ele)
n+=1
for i in range(n):
    print(a1[i],end=' ')